src/main/resources/testchipip/csrc/cospike_impl.cc

```diff
@@ -115,6 +132,7 @@ void cospike_set_sysinfo(char* isa, char* priv, int pmpregions,
     info = new system_info_t;
     // technically the targets aren't zicntr compliant, but they implement the zicntr registers
     info->isa = std::string(isa) + "_zicntr";
+    remove_isa_ext_inplace(info->isa, "xrocket");
```

Comment:
I think a better way of doing this is disabling `xrocket` in the cosim configs.

This can be done with `freechips.rocketchip.rocket.WithNoCease`

Reply:
Sure. I'll do so in the main CY PR.